src/Nox.Solution/Models/Infrastructure/Infrastructure.cs

```diff
@@ -7,23 +8,27 @@ namespace Nox.Solution;
 [Description("Define components pertinent to solution infrastructure here. Examples include persistence, messaging, dependencies and endpoints.")]
 [AdditionalProperties(false)]
 public class Infrastructure : DefinitionBase
-{
-    [Required]
+{    
     // These descriptors should be moved to the class when the generator is fixed
     [Title("The definition namespace for persistance settings pertaining to a Nox solution.")]
     [Description("Defines settings pertinent to solution persistence here. These include database, event source, search and cache servers.")]
     [AdditionalProperties(false)]
-    public Persistence Persistence { get; internal set; } = new();
+    public Persistence? Persistence { get; internal set; }
```

Comment:
Persistence was required only if Infrastructure existed, now we can have Parts of Infrastructure without Persistence